### PR TITLE
Added Switch and case with instantiation of Sandwich

### PR DESCRIPTION
**Pull request by calumforster**
Can we look over this to make sure it fits in with what everyone else is doing?

### CODE REVIEW

File: src/Panini.java [hunk 1]
```diff
@@ -1,5 +1,8 @@
 /**
  * Created by trainee on 18/10/16.
  */
-public class Panini {
+public class Panini extends Sandwich {
```
Comment (calumforster) [inline src/Panini.java]: Made Panini a child of Sandwich

File: src/SandwichOptions.java [hunk 2]
```diff
@@ -2,4 +2,29 @@
...
+            case "Panini":
+                s = new Panini;
+
+
+                break;
+            default:
+                s = null;
+        }
+
+        return s;
```
Comment (calumforster) [inline src/SandwichOptions.java]: This decides if we are returning a bread or Panini  - John will handle the default "null" in his class

File: src/SandwichOptions.java [hunk 3]
```diff
@@ -2,4 +2,29 @@
  * Created by trainee on 18/10/16.
  */
 public class SandwichOptions {
+
+    public Sandwich getSandwich (String sType){
```
Comment (ShannonHolgate) [inline src/SandwichOptions.java]: `sandwichType`

File: src/SandwichOptions.java [hunk 4]
```diff
@@ -2,4 +2,29 @@
...
+
```
Comment (ShannonHolgate) [inline src/SandwichOptions.java]: Unnecessary new lines

File: src/SandwichOptions.java (hunk 3 above)
Comment (ShannonHolgate) [inline src/SandwichOptions.java]: we're expecting the method to be called `getSandwichType`

File: src/SandwichOptions.java [hunk 5]
```diff
@@ -2,4 +2,29 @@
  * Created by trainee on 18/10/16.
  */
 public class SandwichOptions {
+
+    public Sandwich getSandwichType (String sandwichType){
+
+        Sandwich s;
```
Comment (ShannonHolgate) [inline src/SandwichOptions.java]: please use a meaningful name for variables

Comment (ShannonHolgate) [inline src/SandwichOptions.java]: instead of defaulting the switch statement to null should we just set the Sandwich to null here?

**Pull request merged**